src/navigation/task/Complete.js

```diff
@@ -283,9 +283,11 @@ class CompleteTask extends Component {
   }
 
   multipleTasksLabel(tasks) {
+
     return tasks.reduce(
       (label, task, idx) => {
-        return `${label}${idx !== 0 ? ',' : ''} #${task.id}`;
+        const taskIdentifier = task?.metadata?.order_number ? `${task.metadata.order_number}-${task?.metadata?.delivery_position}` : task.id
```

Comment:
I see it a second time, maybe use a util function?

Reply:
where ?